packages/dev/serializers/src/glTF/2.0/Extensions/KHR_lights_punctual.ts

```diff
@@ -111,7 +111,7 @@ export class KHR_lights_punctual implements IGLTFExporterExtensionV2 {
             // glTF lights have variable rotation and constant direction. Therefore,
             // compute a quaternion that aligns the Babylon light's direction with glTF's constant one.
             if (lightType !== KHRLightsPunctual_LightType.POINT) {
-                const direction = babylonNode.direction.normalize();
+                const direction = babylonNode.direction.normalizeToNew();

```

Comment:
This is a good change. It would be even better to avoid allocations completely and use a temp vector with `ToRef` functions. Same with the `Quaternion.RotationAxis` below.

Reply:
This can totally wait for later though, so no need to block the PR for this.

Reply:
No problem, I just pushed changes that fix this.